.github/scripts/build-preview-urls-comment.js

```diff
@@ -33,66 +26,82 @@ export const buildPreviewURLComment = (ref) => {
             color: 'Light',
             scale: 'Medium',
             direction: 'LTR',
+            urlPath: 'spectrum-light-medium-ltr',
+        },
+        {
+            system: 'Spectrum',
+            color: 'Dark',
+            scale: 'Large',
+            direction: 'RTL',
+            urlPath: 'spectrum-dark-large-rtl',
         },
-        { system: 'Spectrum', color: 'Dark', scale: 'Large', direction: 'RTL' },
         {
             system: 'Express',
             color: 'Light',
             scale: 'Medium',
             direction: 'LTR',
+            urlPath: 'express-light-medium-ltr',
+        },
+        {
+            system: 'Express',
+            color: 'Dark',
+            scale: 'Large',
+            direction: 'RTL',
+            urlPath: 'express-dark-large-rtl',
         },
-        { system: 'Express', color: 'Dark', scale: 'Large', direction: 'RTL' },
         {
             system: 'Spectrum-two',
             color: 'Light',
             scale: 'Medium',
             direction: 'LTR',
+            urlPath: 'spectrum-two-light-medium-ltr',
         },
         {
             system: 'Spectrum-two',
             color: 'Dark',
             scale: 'Large',
             direction: 'RTL',
+            urlPath: 'spectrum-two-dark-large-rtl',
         },
     ];
 
-    // Generate preview links for each combination of system, color, scale, and direction
-    previewCombinations.forEach(({ system, color, scale, direction }) => {
-        // Create a unique context string for each combination
-        const context = `${branch}-${system.toLowerCase()}-${color.toLowerCase()}-${scale.toLowerCase()}-${direction.toLowerCase()}`;
-
-        // Add the generated preview link to the array
-        previewLinks.push(`
-- [${system} | ${color} | ${scale} | ${direction}](https://${createHash(
-            context
-        )}--spectrum-wc.netlify.app/review/)`);
-    });
-
-    // Add a high contrast mode preview link
-    previewLinks.push(
-        `
-- [High Contrast Mode | Medium | LTR](https://${createHash(
-            `${branch}-hcm`
-        )}--spectrum-wc.netlify.app/review/)`
+    // Generate preview links for each combination
+    previewCombinations.forEach(
+        ({ system, color, scale, direction, urlPath }) => {
+            const vrtUrl = `${baseUrl}/${prHash}/${urlPath}/review/`;
+
+            previewLinks.push(`
```

Comment:
super nitpicky, but can we push these links inline and then in the end we add the line breaks (`${previewLinks.join('\n')}`)?